phase1/vsphere/do

```diff
@@ -16,6 +16,20 @@ gen() {
 deploy() {
   gen
   terraform apply -state=./.tmp/terraform.tfstate .tmp
+  rm /usr/local/bin/kubectl
+  COMMAND="grep '.phase2.kubernetes_version' ../../.config | cut -d '=' -f2 | tr -d '\"'"
+  KUBERNETES_VERSION=$(eval $COMMAND)
+  echo $KUBERNETES_VERSION
```

Comment:
Since there is an echo command at the bottom mentioning the same. Removing it altogether. Please see the latest change.